regulations/make.py

```diff
@@ -210,7 +211,12 @@ def buildLanguage(args, language):
 
   pdfName = languageData[language]["pdf"]
 
-  html.html(language, buildDir, pdfName + ".pdf", isTranslation=isTranslation, verbose=args.verbose)
+  srcDir = "translations/" + language if isTranslation else "wca-regulations"
+
+  warnings = check_missing.check_guidelines(language, srcDir)
+  if len(warnings) > 0:
+    print warnings.encode('utf-8')
```

Comment:
I see, you're totally right that we'll still need calls to `decode` to get unicode strings, because this is Python 2, and reading files gets you 8-bit strings.

I don't understand that the piping/redirecting issue. Is it possible to write a standalone python script that reproduces it? I'm just curious what's going on.